### DISCUSSION TITLE
agent release please Yilan :)

### DISCUSSION THREAD

**Discussion by bbrendon** [Ylianst/MeshCentral#4659] [General]
Hi @Ylianst

Could we get an agent release? I think a lot of people (including me) are looking for this.
https://github.com/Ylianst/MeshAgent/commit/da840cd37301f46a63e237b536004d9752536ab1

It should close a number of github issues. I think its related to:
https://github.com/Ylianst/MeshCentral/issues/4443
https://github.com/Ylianst/MeshCentral/issues/3397
https://github.com/Ylianst/MeshCentral/issues/4582

For some reason when you use mkpkg the M1 agent doesn't work, but if you do the interactive install it does. The silent install should avoid the need for mpkg which would be an alternative for M1.

**Comment by krayon007** [comment]
Actually, the fixes I made for this don't require the agent to be updated, just the server. I think Ylian just updated the server , so if you use that version, the silent install should work.

**Reply by bbrendon** [reply]
@krayon007 Have you tested it? I tried it on the version that was released on the weekend and didn't have success. I assumed there was something wrong on your end, but since you're saying thats not the case, I'll look at it closer.

**Reply by silversword411** [reply]
Commit was in the agent repo, so thought it needed an agent release as well.

https://github.com/Ylianst/MeshAgent/commit/da840cd37301f46a63e237b536004d9752536ab1

**Reply by krayon007** [reply]
Let me pull from the latest server, and verify everything is in there.

**Reply by krayon007** [reply]
Ok you're right, I was an idiot. I made a bunch of changes to the server repo's version of the installer for other things, but the specific change for the silent install issue was missing... I just merged that in now... So the next time the server gets updated, it will be included.
